bin/analyze-all-exercises-in-docker.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e # Make script exit when a command fail.
+set -u # Exit on usage of undeclared variable.
+# set -x # Trace what gets executed.
+set -o pipefail # Catch failures in pipes.
+
+# Command line argument
+elm_repo_dir=$(realpath $1)
+
+# build docker image
+docker build --rm -t elm-analyzer .
```

Comment:
This should match the image name in the GHA workflow:

```suggestion
docker build --rm -t exercism/elm-analyzer .
```